cirq/google/devices/known_devices.py

```diff
@@ -223,3 +224,37 @@ def _json_dict_(self):
 BRISTLECONE_PROTO = create_device_proto_from_diagram(_BRISTLECONE_GRID,
                                                      [gate_sets.XMON],
                                                      _DURATIONS_FOR_XMON)
+
+_SYCAMORE_GRID = """
+-----AB---
+----ABCD--
+---ABCDEF-
+--ABCDEFGH
+-ABCDEFGHI
+ABCDEFGHI-
+-CDEFGHI--
+--EFGHI---
+---GHI----
+----I-----
+"""
+
+_SYCAMORE_DURATIONS_PICOS = {
+    'xy': 25_000,
+    'xy_half_pi': 25_000,
+    'xy_pi': 25_000,
+    'fsim_pi_4': 32_000,
+    'inv_fsim_pi_4': 32_000,
+    'syc': 12_000,
+    'z': 0,
+    'meas': 1_000_000,
+}
+
+SYCAMORE_PROTO = create_device_proto_from_diagram(
+    _SYCAMORE_GRID,
+    [gate_sets.SQRT_ISWAP_GATESET, gate_sets.SYC_GATESET],
+    _SYCAMORE_DURATIONS_PICOS,
+)
+
+Sycamore = SerializableDevice.from_proto(
```

Comment:
OK calling this Sycamore...  Maybe at some point we need to move to calling these prototypes, since the real numbers and grids will come from engine?

Reply:
Possibly.  We need to think this through a bit.  This stays with the naming system with BristleCone and FoxTail.